Task3/js/main.js

```diff
@@ -0,0 +1,45 @@
+//1.
+//listen for window resize event
+window.addEventListener('resize', function(event){
+  var newWidth = window.innerWidth;
+  var newHeight = window.innerHeight;
+  console.log(newWidth)
+  console.log(newHeight)
+});
+
+//2.
+//Change URL params
+// Construct URLSearchParams object instance from current URL querystring.
+function setUrlParameter(param_name, param_value){
+  let theURL              = new URL('https://dummy.com');
+  theURL.searchParams.set(param_name, param_value);
+  console.log(theURL.href)
+}
+
+//3.
+//Change hash URL params
+// Construct URLSearchParams object instance from current URL querystring.
+function setUrlHashParameter(param_name, param_value){
+  let theURL              = new URL('https://dummy.com');             // create dummy url
```

Comment:
You should init empty `new URLSearchParams()` and not manipulate with absolute url.